src/components/ColorBarLegend/ColorBarItem.tsx

```diff
@@ -39,7 +39,7 @@ const colorBarItemStyle = (theme: Theme) => ({
 const useItemStyles = makeStyles((theme: Theme) => ({
```

Comment:
makeStyles is also imported from @mui/styles which is now legacy, so may  be we can get rid of makeStlyes and adapt styled or sx

Reply:
Yes, of course. I will adapt all the components. I have not yet started this.